0000-named-tracers.md

```diff
@@ -0,0 +1,67 @@
+# Named Tracers 
+
+_Creating tracers using a factory mechanism and naming those tracers in accordance with the (logical) components being traced._
+
+## Suggested reading
+
+* [Proposal: Tracer Components](https://github.com/open-telemetry/opentelemetry-specification/issues/10)
+* [Global Instance discussions](https://github.com/open-telemetry/opentelemetry-specification/labels/global%20instance)
+
+## Motivation
+
+Instead of having a global tracer singleton, every library needs to create its own tracer, which also has a proper name that identifies the library / logical component that uses this tracer.
+
+This proposal originates from an `opentelemetry-specification` proposal on [components](https://github.com/open-telemetry/opentelemetry-specification/issues/10), but the term "components" was quickly found to being too generic and possibly misleading, since it is already used frequently in differing contexts and connotations.
+
+The purpose of _Named Tracers_ is the ability to allow for a more fine-grained control of the amount of tracing data being generated by a library or application. Associating a name (e.g. _"io.opentelemetry.contrib.mongodb"_) with a tracer allows for a logical coupling of this tracer and a certain technology or library (like _MongoDB_ in this example).
+
+By using *Named Tracers* a library can:
+* Identify the spans it created, by automatically setting e.g. a `component` (usually the tracer's name) on each newly created span.
```

Comment:
While this is certainly also a possibility, I would leave this out of the RFC for now. Because this would incur new discussion about new API's (e.g. a TracerBuilder that allows setting of these components or setter methods in a Tracer to modify those.)

Reply:
Addressed in follow-up commit.